src/pudl/metadata/fields.py

```diff
@@ -871,6 +875,10 @@
         "type": "string",
         "description": "EIA short code indicating the type of firing used by this boiler.",
     },
+    "flue_id_eia": {
+        "type": "string",
+        "description": "test",
```

Comment:
This seems to be a leftover draft. Should likely be "The flue identification number provided by EIA" or something similar.